### ISSUE TITLE
Group by files by day

### ISSUE THREAD

**Issue by vshcherb** [osmandapp/Osmand#7667]
https://github.com/osmandapp/Osmand/pull/7663

**Comment by jensMF** [comment]
Could you go into more detail? Maybe I could implement it?

**Comment by jensMF** [comment]
@vshcherb I have a strange issue I don't know where would be the right place to ask.
When I build OsmAnd (the actual Master), buildings and other polygons from offline-maps does not get rendered. Do you know how I could solve this?

**Comment by vshcherb** [comment]
you need to have native libararies, you can either build it https://osmand.net/help-online#dev-environment or copy native libs in libs folder from apk

**Comment by jensMF** [comment]
Ah, thank you. I implemented the daily directories as direct subdirectories to rec but first thought about implementing them as subdirectories of the monthly directories. But since they where not shown as subdirectories of the monthly directories in the "My Tracks" area, I did it as mentioned above. If they where shown as subdirectories of the monthly folders, I would prefer to implement daily directories as subdirectories of monthly directories.
